extensions/azurePublish/src/services/appRegistrationService.ts

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { throwNotImplementedError } from './throwNotImplementedError';
+
+/* Use Graph Client
+https://www.npmjs.com/package/@microsoft/microsoft-graph-client
+*/
+export const createAppRegistrationService = (graphToken: string) => {
+  const checkExistence = async (displayName: string) => {
```

Comment:
Nit: For the initial check in, I would have the methods either have the arguments defined for all the services or have no arguments defined.